pom.xml

```diff
@@ -48,14 +48,15 @@
   </distributionManagement>
 
   <properties>
-    <maven.compiler.source>17</maven.compiler.source>
-    <maven.compiler.target>17</maven.compiler.target>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <spotless.version>2.43.0</spotless.version>
     <jackson.version>2.17.2</jackson.version>
     <slf4j.version>2.0.13</slf4j.version>
     <okhttp.version>4.12.0</okhttp.version>
+    <autograder.version>0.5.41</autograder.version>
```

Comment:
We should make a new release before merging this.

Reply:
Yes, I'd say the changes are worth of 0.6.0, but let's wait until the current exercise's grading is completed